buildo/react-components
# #596: [KitchenSink] Various UI fixes (closes #596)

Issue #596

## Test Plan

### tests performed
- `npm run`
- Check over every UI fix listed in the issue


- left sidebar hover color:
![image](https://cloud.githubusercontent.com/assets/925635/18435739/9afd09b4-78fd-11e6-9266-a9654a9b2d79.png)

- button size in default example is medium
![image](https://cloud.githubusercontent.com/assets/925635/18433642/815d0f56-78f1-11e6-8cc5-d6de2f2f38b6.png)

- IconButton icon replaced with a new one
![image](https://cloud.githubusercontent.com/assets/925635/18433657/9c8f630a-78f1-11e6-8bdc-57b3114bf306.png)

- new dropdown arrows
![image](https://cloud.githubusercontent.com/assets/925635/18433674/b064e8e6-78f1-11e6-8184-bcaf94d5b153.png)

- Icon example
![image](https://cloud.githubusercontent.com/assets/925635/18433678/baa97c54-78f1-11e6-8b3f-ad65032f293a.png)

- open/close arrows in CollapsableSection swapped
![image](https://cloud.githubusercontent.com/assets/925635/18433710/daa44dfe-78f1-11e6-824e-918518ac42f9.png)

- badges are using the right font
![image](https://cloud.githubusercontent.com/assets/925635/18433616/51e8ed62-78f1-11e6-869d-3bbea9cdd66d.png)

- MoreOrLess example image has higher quality
![image](https://cloud.githubusercontent.com/assets/925635/18433753/f2213abe-78f1-11e6-9b39-90ff1802e54f.png)

- `active` color in navbar changed
![image](https://cloud.githubusercontent.com/assets/925635/18435647/2cdd68ca-78fd-11e6-9671-c05cc1418f1e.png)

- Modal has rounded borders **(fix to the component style itself, shouldn't be breaking)**
![image](https://cloud.githubusercontent.com/assets/925635/18435688/6d86448c-78fd-11e6-9ca6-1a35e981ee60.png)

- ::selection style
![image](https://cloud.githubusercontent.com/assets/925635/18435762/c04d3f22-78fd-11e6-8537-c48bbfe656ca.png)
